api/Helpers/ClaimsTypes.cs

```diff
@@ -16,6 +16,7 @@ public class CustomClaimTypes
         public const string PreferredUsername = "preferred_username";
         public const string JasperPermission = "jasper_permission";
         public const string JasperRole = "jasper_role";
+        public const string UserId = "jasper_user_id";
```

Comment:
Still seeing naming conventions that should be updated.  For example, these ones: 
```suggestion
        public const string Permission = "permission";
        public const string Role = "role";
        public const string UserId = "user_id";
```